message_definitions/v1.0/common.xml

```diff
@@ -856,6 +856,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="188" name="MAV_CMD_DO_LAND_REJOIN" hasLocation="true" isDestination="false">
+        <description>Mission command to perform a landing. This is used as a marker in a mission to tell the autopilot that is can rejoin a mission at any point after this marker and before a landing or MAV_CMD_DO_LAND_START, the autopilot will then rejoin the mission at the closet point following it to the landing.</description>
```

Comment:
```suggestion
        <description>Mission command to perform a landing. This is used as a marker in a mission to tell the autopilot that it can rejoin a mission at any point after this marker and before a landing or MAV_CMD_DO_LAND_START. The autopilot will then rejoin the mission at the closet point following it to the landing.</description>
```